glean-core/python/glean/config.py

```diff
@@ -3,9 +3,49 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 
+import dataclasses
+from typing import Optional
+
+
+# The default server pings are sent to
+DEFAULT_TELEMETRY_ENDPOINT = "https://incoming.telemetry.mozilla.org"
+
+
+# The default UserAgent string
+def get_default_user_agent():
+    # Need to use a function here to avoid a cyclical import problem
+    from glean import __version__
+
+    return f"Glean/{__version__} (Python)"
```

Comment:
This is interesting. For other platforms, we used `Android` or `ios`. Here we're using a language, `Python`. This can indeed run on any platform, including `Android`, so it's probably the best choice.

Feels a bit odd, though :)

Reply:
We could do both.  What about `Glean/{__version__} (Python on {sys.platform})`.  sys.platform gives: `linux`, `darwin` or `win32`.